internal/policies/fetcher.go

```diff
@@ -63,6 +63,30 @@ func (f *Fetcher) GetPoliciesForANamespace(namespace string) ([]policiesv1.Polic
 	return filteredPolicies, skippedNum, nil
 }
 
+func (f *Fetcher) getClusterAdmissionPolicies() ([]policiesv1.ClusterAdmissionPolicy, error) {
+	policies := &policiesv1.ClusterAdmissionPolicyList{}
+	err := f.client.List(context.Background(), policies)
+	if err != nil {
+		return nil, err
+	}
+	return policies.Items, nil
+}
+
+func (f *Fetcher) GetClusterAdmissionPolicies() ([]policiesv1.Policy, int, error) {
+	clusterAdmissionPolicies, err := f.getClusterAdmissionPolicies()
+	if err != nil {
+		return nil, 0, err
+	}
+	policies := []policiesv1.Policy{}
+	for _, policy := range clusterAdmissionPolicies {
+		policy := policy
+		policies = append(policies, &policy)
+	}
+	filteredPolicies := f.filter(policies)
```

Comment:
Unrelated with this PR. I was having a hard time figuring out why we should be filtering the cluster policies. In the end it makes sense, but there's a layer of indirection introduced by having `f.filter` be a function pointer.

Do we really need that? Can't we just invoke the `filterAuditablePolicies` function? That would make the code more readable IMHO

Reply:
in the future, we could use the filter function to cribe for policies, in addition or regardless of their `spec.ContextAwareResources`. For example we could try to run all policies against all resources: if the resource is listed in `spec.ContextAwareResources`, we evaluate as usual. If it is not, we do our best, and if we get an AdmissionReview with an error back, we ignore the result. This would allow us to naively audit as much things as possible.

But agree, for now, I can't find a utility and it doesn't make the code more readable.

Reply:
Let's leave the code as it is in the meantime. The complexity is not so big